microsoft/testsuites/network/sriov.py

```diff
@@ -1,29 +1,39 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 from pathlib import Path
-from typing import Any, Dict, List, cast
+from typing import Any, cast
 
 from assertpy import assert_that
 
 from lisa import (
     Environment,
     Logger,
-    Node,
     RemoteNode,
     TestCaseMetadata,
     TestSuite,
     TestSuiteMetadata,
+    node_requirement,
     schema,
     search_space,
     simple_requirement,
 )
 from lisa.features import NetworkInterface, SerialConsole
-from lisa.nic import NicInfo, Nics
+from lisa.features import Sriov as SriovNetwork
```

Comment:
If there is a conflict, you can import it like `from lisa import features`, and use it like `features.Sriov`

Reply:
updated